website_sale_product_pack/__manifest__.py

```diff
@@ -4,7 +4,7 @@
     "name": "Website Sale Product Pack",
     "category": "E-Commerce",
     "summary": "Compatibility module of product pack with e-commerce",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.1.0",
```

Comment:
Please don't change the version manually. It will be done by the bot.

Reply:
@pedrobaeza Sorry! it is already corrected